when-shit-hits-the-fan.adoc

```diff
@@ -332,9 +332,9 @@ The "value out" in this block #74638 is quite strange:
 92233720368.54277039 BTC?  Is that UINT64_MAX, I wonder?
 ____
 
-Apparently, the two int64 (not uint64 as Garzik pondered) outputs' sum
+Apparently, the two int64 (not uint64, as Garzik supposed) outputs' sum
```

Comment:
Apparently, there was a bug where the two int64 ...

Reply:
This also helps making things clearer, thanks!